src/test/resources/assets/music_disc_test/sounds.json

```diff
@@ -0,0 +1,13 @@
+{
+  "test_sound_event": {
```

Comment:
Data generators would be nice for all the jsons

Reply:
Should I create a new data provider for this situation in this PR or should I open up a new one with the specified addition?

Reply:
Make a new PR for the added data gen.